example/bayesian-methods/bdk_demo.py

```diff
@@ -156,42 +156,42 @@ def get_toy_sym(teacher=True, teacher_noise_precision=None):
     return net
 
 
-def dev():
-    return mx.gpu()
+def dev(xpu = None):
```

Comment:
No need to have white spaces before and after `=`. Also, `xpu` is a little confusing for me as its value should be either `gpu` or `cpu` according to the name. But here in this case it equals to gpu numbers.